scripts/temples.js

```diff
@@ -0,0 +1,19 @@
+function toggleMenu() {
```

Comment:
Remove the title change reference in this function. The title tag is in the head, so you cannot style it, especially for this hamburger button.

```javascript
function toggleMenu() {
    const menu = document.getElementById('menu');

    if (menu.style.display === 'flex') {
        menu.style.display = 'none';
    } else {
        menu.style.display = 'flex';
    }
}
```